web/gds-user-ui/src/modules/dashboard/organization/organizationType.ts

```diff
@@ -1,30 +1,41 @@
 export interface Organization {
-    id: string;
-    name: string;
-    domain?: string;
-    createdAt?: string;
-    refreshToken?: string;
+  id: string;
+  name: string;
+  domain?: string;
+  createdAt?: string;
+  refreshToken?: string;
+}
+
+export interface organizationResponse {
+  count: number;
+  organizations: Organization[];
+  page: number;
+  page_size: number;
 }
 
 export type NewOrganization = Pick<Organization, 'id' | 'name' | 'domain'>;
 
 export type OrganizationMutation = {
-    createOrganization(organization: NewOrganization): void;
-    reset(): void;
-    organization?: Organization;
-    hasOrganizationFailed: boolean;
-    wasOrganizationCreated: boolean;
-    isCreating: boolean;
-    errorMessage?: any;
+  createOrganization(organization: NewOrganization): void;
+  reset(): void;
+  organization?: Organization;
+  hasOrganizationFailed: boolean;
+  wasOrganizationCreated: boolean;
+  isCreating: boolean;
+  errorMessage?: any;
 };
 
 export type OrganizationQuery = {
-    getAllOrganizations(): void;
-    reset?(): void;
-    organizations?: Organization[];
-    hasOrganizationFailed: boolean;
-    wasOrganizationFetched: boolean;
-    isFetching: boolean;
-    errorMessage?: any;
+  getAllOrganizations(): void;
+  reset?(): void;
+  organizations?: any;
```

Comment:
Would you mind adding the right type instead of `any` ?

Reply:
Sure thanks. But s story is planned to migrate all the `any` type definitions. it has been added when the backend is not ready yet and was the best way to go quickly.